bzr.go

```diff
@@ -64,6 +65,12 @@ func (s BzrRepo) Vcs() Type {
 
 // Get is used to perform an initial clone of a repository.
 func (s *BzrRepo) Get() error {
+
+	basePath := path.Join(s.LocalPath(), "..")
```

Comment:
Instead of `path.Join` this should be `filepath.Join` so that it handles cross platform directory separators correctly.